addons/website_payment/static/tests/tours/donation.js

```diff
@@ -64,6 +65,91 @@ registerWebsitePreviewTour(
             content: "Verify that other_amount is not checked",
             trigger: ":iframe input#other_amount:not(:checked)",
         },
+        {
+            content: "Click on the custom amount radio button",
+            trigger: ":iframe input#other_amount",
+            run: "click",
+        },
+        {
+            content: "Verify that other_amount is checked",
+            trigger: ":iframe input#other_amount:checked",
+        },
+        {
+            content: "Submit the donation form",
+            trigger: ":iframe button[name='o_payment_submit_button']",
+            run: "click",
+        },
+        {
+            content: "Check warning message is display",
+            trigger: ":iframe .o_donation_payment_form:has(p#warningMessageId)",
+        },
+        {
+            content: "Enter an amount less than the minimum value",
+            trigger: ":iframe input#other_amount_value",
+            run: "edit 1",
+        },
+        {
+            content: "Verify the display of a minimum value warning message",
+            trigger: ":iframe p#warningMinMessageId:contains('The minimum donation amount is $5')",
+        },
+        {
+            content: "Click on the first radio button",
+            trigger: ":iframe input[name='o_donation_amount']:first-child",
+            run: "click",
+        },
+        {
+            content: "Ensure the custom amount value is cleared",
+            trigger: ":iframe input[name='o_donation_amount']:first-child",
+            run: () => {
+                const iframe = document.querySelector("iframe").contentDocument;
+                const warningEl = iframe.querySelector("#warningMinMessageId");
+                const customAmountEl = iframe.querySelector("#other_amount_value");
+                if (!warningEl.classList.contains("d-none") || customAmountEl.value !== "") {
+                    console.error("Custom amount should be cleared.");
+                }
+            },
+        },
+        {
+            content: "Click on home page",
+            trigger: ":iframe span[data-oe-model='website.menu']",
+            run: "click",
+        },
+        ...clickOnEditAndWaitEditMode(),
+        {
+            content: "Click on custom donation button",
+            trigger: ":iframe .s_donation_donate_btn",
+            run: "click",
+        },
+        {
+            content: "Click on selection button",
+            trigger: "we-customizeblock-option.snippet-option-Donation we-toggler",
+            run: "click",
+        },
+        {
+            content: "Change button to slider",
+            trigger: "we-button[data-name='slider_opt']",
+            run: "click",
+        },
+        ...clickOnSave(),
+        {
+            content: "Select any button after prefilled buttons",
```

Comment:
- what does it mean ? Which buttons are you talking about ?
- `s_donation_btn_description` is a `<div>` (and it's not after "prefilled buttons")